book/02-git-basics/sections/viewing-history.asc

```diff
@@ -264,7 +265,7 @@ In <<limit_options>> we'll list these and a few other common options for your re
 [cols="2,4",options="header"]
 |================================
 | Option                | Description
-| `-(n)`                | Show only the last n commits
+| `-(n)`                | Show only the last n commits (really a shortcut for `-n (n)`, so `-5` and `-n 5` are equivalent)
```

Comment:
I didn't know there were technical issues with longer lines. I don't really
know what program you are putting this through to do the final formatting.

The only reason for the suggestion was that I was trying to connect the
-(n) with what feels like the more "real" command-line option of "-n (n)".

On Fri, Aug 5, 2016 at 4:01 PM, Ben Straub <notifications@github.com> wrote:

> In book/02-git-basics/sections/viewing-history.asc
> <https://github.com/progit/progit2/pull/630#discussion_r73765824>:
>
> > @@ -264,7 +265,7 @@ In <<limit_options>> we'll list these and a few other common options for your re
> >  [cols="2,4",options="header"]
> >  |================================
> >  | Option                | Description
> > -| `-(n)`                | Show only the last n commits
> > +| `-(n)`                | Show only the last n commits (really a shortcut for `-n (n)`, so `-5` and `-n 5` are equivalent)
>
> I worry a bit about the line length in the table cell here. Some renderers
> might do a terrible job of wrapping an overlong line. Can we change it to
> this?
>
> | `-(n)` or `-n (n)`    | Show only the last n commits
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/progit/progit2/pull/630/files/2cb2f4600180163db5ac70f1f440e28cbf88c975#r73765824>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABmTbs54e2b4qiRpTXD7kkuZNPEoZQlbks5qc7LKgaJpZM4JYCxK>
> .
>